ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -115,7 +115,8 @@ enum CocoaPodUtils {
     }
 
     // Run pod install on the directory that contains the Podfile and blank Xcode project.
-    let result = Shell.executeCommandFromScript("pod _1.8.4_ install", workingDir: directory)
+    checkCocoaPodsVersion(directory: directory)
+    let result = Shell.executeCommandFromScript("pod install", workingDir: directory)
```

Comment:
AFAIK just `pod install` is not necessarily uses Bundler version (it depends on other Ruby environment settings). `bundle exec pod install` should be more environment-independent version.